NewForce-Cohort-9/tabloidmvc-josh-jacobs-jingleheimer-schmidt
# Index View for Tag Management Complete

## Description
Added logged in admin Index view for Tag Management link in navbar. This lists all tags in the database alphabetically, as requested in [ticket #13](https://trello.com/c/rRHKX46x/13-13-view-all-tags)

## Testing Instructions
- `git add` and `git commit` whatever you're working on your branch
- `git checkout main`
- `git fetch --all`
- `git checkout rev-tag-index`
- If you don't see the most recent version of my files, `git pull origin rev-tag-index`
- If you haven't created the Tabloid MVC database, please do so.
- Click the execute with debugger solid green button to launch the app
- In browser, please go to Login on the nav bar and log in with this email: [admin@example.com](mailto:admin@example.com)
- This should redirect you back to the home page.
- Click on the link in the Navbar titled "TAG MANAGEMENT"
- This should take you to the /tag URL and display an alphabetized list of all tags from the database:
   - C#
   - Cyclopean Terrors
   - Family
   - JavaScript

## Checklist:
